Submission.java

```diff
@@ -133,6 +133,83 @@ public int getScriptCount()
     }
 
     /**
+     * Get script count for stage.
+     *
+     * @return count 
+     */
+    public int getScriptCountForStage()
+    {
+        return getCountForStage("scripts");
+    }
+
+    /**
+     * Get variable count for stage.
+     *
+     * @return count 
+     */
+    public int getVariableCountForStage()
+    {
+        return getCountForStage("variables");
+    }
+
+    /**
+     * Get list count for stage.
+     *
+     * @return count 
+     */
+    public int getListCountForStage()
+    {
+        return getCountForStage("lists");
+    }
+    
+    /**
+     * Get script comment count for stage.
+     *
+     * @return count 
+     */
+    public int getScriptCommentCountForStage()
+    {
+        return getCountForStage("scriptComments");
+    }
+
+    /**
+     * Get sound count for stage.
+     *
+     * @return count 
+     */
+    public int getSoundCountForStage()
+    {
+        return getCountForStage("sounds");
+    }
+
+    /**
+     * Get costume count for stage.
+     *
+     * @return count 
+     */
+    public int getCostumeCountForStage()
+    {
+        return getCountForStage("costumes");
+    }
+    
+    /**
+     * Get count for stage.
+     * 
+     * @param item 
+     * @return count 
+     */
+    private int getCountForStage(String item)
+    {
+        JSONArray items = 
+            FileUtils.getJSONArrayAttribute(jsonObj, item);
+        if (items != null)
+        {
+            return (int) items.size();
+        }
+        return 0;
+    }
+
+    /**
      * Get array of sprites.
      * Unchecked warnings are suppresed because JSONArray does not
```

Comment:
@KaraBeason I'm sorry - this one was actually me! Too bad checkstyle doesn't check spelling too! :) Thanks for reviewing this!